packages/pino/src/index.ts

```diff
@@ -17,21 +17,25 @@ export interface Options extends ClientOptions {
 
 export default async function axiomTransport(options: Options) {
   const axiom = new Axiom(options);
-
   const dataset = options.dataset;
 
   return build(
     async function (source: any) {
       for await (const obj of source) {
         const { time, level, ...rest } = obj;
-
         const event = {
           _time: time,
           level: mapLogLevel(level),
           ...rest,
         };
 
-        axiom.ingest(dataset, event);
+        try {
+          await axiom.ingest(dataset, event);
```

Comment:
do we want to await here? seems like not blocking is probably better, and how it was done before?

Reply:
@c-ehrlich, if we remove the `await` and collect the promises returned by each ingestion? as it's now non-blocking

Reply:
@thecraftman 👍

Reply:
@c-ehrlich fixed